src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -1042,6 +1043,123 @@ unpack_ncp_counters_mle(const uint8_t *data_in, spinel_size_t data_len, boost::a
 	return ret;
 }
 
+static int
+unpack_coex_metrics(const uint8_t *data_in, spinel_size_t data_len, boost::any& value, bool as_val_map)
+{
+	std::list<std::string> result_as_string;
+	ValueMap result_as_val_map;
+	int ret = kWPANTUNDStatus_Ok;
+	spinel_ssize_t len;
+	bool stopped;
+	uint32_t numGrantGlitch;
```

Comment:
not sure if we should unify to use snake case.

Reply:
I have updated the name to keep the code style the same.